patch/secure_boot_kernel_config.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Note: this script was created because there is a problem when changing the kernel config
+# values that requires in the Secure Boot feature when using patch/kconfig-inclusions (sonic flow to modify kernel flags).
+# So, when this problem will be resolved, this script should be removed and used the kconfig-inclusions.
+
+usage() {
+    cat <<EOF
+$0: # Display Help
+$0 <PEM_CERT>
+Script is modifying kernel config file to support system trusted key with custom certificate.
+Note: The signature algorithem used will be SHA512.
+
+Parameters description:
+PEM_CERT                             public key (pem format). Key to be store in kernel.
+
+Run example:
+bash secure_boot_kernel_config.sh cert.pem
+EOF
```

Comment:
Can’t normal `echo` be used?

Reply:
its also common to add some "Usage example:" as well